smart/plugin.go

```diff
@@ -166,6 +177,10 @@ func (sc *SmartCollector) GetMetricTypes(_ plugin.ConfigType) ([]plugin.MetricTy
 
 //GetConfigPolicy returns a ConfigPolicy
 func (p *SmartCollector) GetConfigPolicy() (*cpolicy.ConfigPolicy, error) {
-	c := cpolicy.New()
-	return c, nil
+	cp := cpolicy.New()
+	rule, _ := cpolicy.NewStringRule("proc_path", false, "/proc")
+	node := cpolicy.NewPolicyNode()
+	node.Add(rule)
+	cp.Add([]string{nsVendor, nsClass, PluginName}, node)
```

Comment:
Supporting a global config calls for example configuration - please consider adding a configuration snippet to README. This way users could start using the plugin immediately.